### DISCUSSION TITLE
Question about how to update the model

### DISCUSSION THREAD

**Discussion by PouyaFakhari** [lllyasviel/Fooocus#2912] [Q&A] [answered]
Hello to all , I'm kind of new in this field, I wanted to know if I install and run this model on my windows system, does it work completely offline or does it need to be connected to the internet, and about the update, if there is an update for the model, can It update automatically it self? i mean Every time I run it, it checks for an update and if there is, does it update? I am new in this field, thank you

**Comment by mashb1t** [comment, marked as answer]
Fooocus is not a model, but a tool which uses models (checkpoints / LoRAs). It updat s itself on launch, so you'll always have the latest available version if you don't tinker with the code. Fooocus needs an internet connection to download everything the first time you use each feature, but never afterwards except for updating itself on boot, but if not possible it will just skip this process.
Hope this answers your questions.

**Reply by PouyaFakhari** [reply]
Thanks for the answer❤️
